README.md

```diff
@@ -1,6 +1,9 @@
 <p align="center">
     <a href="https://github.com/yiisoft" target="_blank">
-        <img src="https://github.com/yiisoft.png" height="100px">
+        <img src="https://yiisoft.github.io/docs/images/yii_logo.svg" height="100px" alt="Yii">
+    </a>
+    <a href="https://swagger.io/" target="_blank">
+        <img src="https://static1.smartbear.co/swagger/media/assets/images/swagger_logo.svg" height="80px" alt="Swagger">
```

Comment:
It looks strange.

![image](https://github.com/yiisoft/yii-swagger/assets/525501/0be7aef3-1756-42e1-bbbb-d389e3e6f968)

Reply:
logos...dark or light mode problem. I need guidance on how to resolve this, or else remove

Reply:
removed.   done